rust/main/chains/hyperlane-sealevel/src/interchain_gas.rs

```diff
@@ -168,13 +168,24 @@ impl SealevelInterchainGasPaymasterIndexer {
             gas_amount: gas_payment_account.gas_amount.into(),
         };
 
-        let log_meta = self
-            .interchain_payment_log_meta(
-                U256::from(sequence_number),
-                &valid_payment_pda_pubkey,
-                &gas_payment_account.slot,
-            )
-            .await?;
+        // let log_meta = self
+        //     .interchain_payment_log_meta(
+        //         U256::from(sequence_number),
+        //         &valid_payment_pda_pubkey,
+        //         &gas_payment_account.slot,
+        //     )
+        //     .await?;
+
+        let log_meta = LogMeta {
+            address: self.igp.program_id.to_bytes().into(),
+            block_number: gas_payment_account.slot,
+            // TODO: get these when building out scraper support.
+            // It's inconvenient to get these :|
+            block_hash: H256::zero(),
+            transaction_id: H512::zero(),
+            transaction_index: 0,
+            log_index: U256::zero(),
```

Comment:
can you make this `sequence_number`? otherwise the log metas aren't unique and IGP payments will be viewed as duplicates - i think this was the cause of https://discord.com/channels/935678348330434570/1306616756361756683

Reply:
Fixed